src/software/geom/angle.h

```diff
@@ -178,6 +178,15 @@ class Angle final
      */
     constexpr Angle minDiff(const Angle &other) const;
 
+    /**
+     * Returns the smallest possible rotational difference from this angle to another angle.
+     *
+     * @param to the second angle.
+     *
+     * @return the angle between this Angle and other, in the range [-π, π].
+     */
+    constexpr Angle minSignedDiff(const Angle &to) const;
```

Comment:
I believe this is what `clamp` already does. If you look at the implementation of `minDiff` as well, it just takes the absolute value of the clamped angle.

Reply:
bump @mkhlb